displaylink.spec

```diff
@@ -167,6 +167,9 @@ chmod +x %{buildroot}%{_prefix}/lib/systemd/system-sleep/displaylink.sh
 %systemd_postun_with_restart displaylink.service
 
 %changelog
+* Sat May 30 2020 Mitya Eremeev <mitossvyaz@mail.ru>
+- fix typo in kernel package name
+- tested package in CentOS 8
```

Comment:
You should add `1.7.0-2` to the end of the line with the date and author information.